### ISSUE TITLE
Adjustment of create/import dialog - name and description fields

### ISSUE THREAD

**Issue by tomas-knap** [UnifiedViews/Core#386]
When I try to create/import DPU template, I got dialog where I can define name, description and select JAR.

![screen shot 2015-05-07 at 15 30 59](https://cloud.githubusercontent.com/assets/3014917/7516230/17d4ea0c-f4ce-11e4-93c7-68cc01ef24cd.png)

This should be adjusted based on the features in https://github.com/UnifiedViews/Core/pull/358

Solution: 
When creating/importing DPU template, user cannot specify custom name, description. 
1) So the dialog has to be adjusted, so that it does not allow definition of name/description.

**Comment by skrchnavy** [comment]
@tomas-knap should this be implemented as a part of #358 pull request?

**Comment by skrchnavy** [comment]
Pull Request #358 merged so not included there,

**Comment by tomas-knap** [comment]
Yes, separately. This should be done for 2.1 as it is for 1h

**Comment by peterklimo** [comment]
#cc4c10e  in feature/adj_cr_imp_dlg

**Comment by tomas-knap** [comment]
Adjustment of GUI ok, but 

1. The following line was removed, which causes that I can import the same DPU with the default name several times

```
if (name == null || parent.getName().equals(name)) { // the same jarFileName and name
-                throw new DPUCreateException(Messages.getString("DPUModuleManipulator.dpu.already.exists", name, newDpuFileName));
-            }
```

![screen shot 2015-06-24 at 11 52 27](https://cloud.githubusercontent.com/assets/3014917/8327287/84517286-1a67-11e5-953c-e29d7a0bba9e.png)

**Comment by tomas-knap** [comment]
```
DPUTemplateRecord dpuWithSameName = dpuFacade.getByName(name);
-            if (dpuWithSameName != null && newDpuFileName.equals(dpuWithSameName.getJarName())) {
-                throw new DPUCreateException(Messages.getString("DPUModuleManipulator.dpu.name.already.exists", newDpuFileName, name));
-            }
```

This condition is basically checking that the name used for the new DPU template is not already used for a different DPU template with the same JAR file. So it cannot happen that certain 3rd level DPU template with custom name is equal to newly imported localized DPU template. So it should not be completely removed.

**Comment by peterklimo** [comment]
fixed by #d7a96c9

**Comment by skrchnavy** [comment]
Fixed in Pull request #429 and merged into develop.
Cherry picked into release branch in commits 79b82cfbaa1cc09f1aa8a577b9f2aed075fd9ee0 and 02e14611921a1eaa4518fb6ab3643497820ab738
